mpf/platforms/fast/fast.py

```diff
@@ -567,7 +568,10 @@ def configure_light(self, number, subtype, config, platform_settings) -> LightPl
 
             elif int(parts[0]) > 255:
                 # EXP LED in int form, which is how "previous:" values are calculated
-                this_led_number = hex(int(parts[0]))[2:]
+
+                raw_hex_string = hex(int(parts[0]))  # lowercase and prefixed with "0x"
```

Comment:
Minor nit, if you were to keep the `[2:]` here then you wouldn't need to have both it and the ` - 2` in the next line, but it's not a huge deal either way.

Reply:
Oh yeah I had a separate variable at first for hex(int(.)) but got a linter error on too many local vars. I think I added a linter exception above, so I could take this back out again

Reply:
No wait I confused myself - too many variations of this same line. I think my thought for this version was that something like "0x9000beef" made more sense as a value I would pass to a cleanup util function than, but then none quite matched what I wanted and I had to strip the 0x here anyway.